### DISCUSSION TITLE
Multi Object & mobilnet

### DISCUSSION THREAD

**Discussion by hyunaseo** [STVIR/pysot#468] [Q&A] [answered]
Thanks for great work!!!!

I have several questions for practical using. 
1. siamrpn_mobilev2 means the model uses mobilenet V2 for backbone?
If it's right, then can I use this model on ios devices?

2. I want to apply that model to situations where the system tracks multiple objects. 
Isn't there any other possible way but to allocate one gpu per object?

It would be very helpful if you could give me an answer. Thank you.

**Comment by ZhiyuanChen** [comment, marked as answer]
> 1. siamrpn_mobilev2 means the model uses mobilenet V2 for backbone?

Yes

>    If it's right, then can I use this model on ios devices?

Yes

> 2. I want to apply that model to situations where the system tracks multiple objects.
>    Isn't there any other possible way but to allocate one gpu per object?

You could always run multiple algorithms on the same GPU as long as you have enough memory. Though I would suggest you try some mot algorithms instead.

**Reply by hyunaseo** [reply]
Thanks for answering. 
Actually I'm quite unfamiliar to this kind of network, so if it's ok, would you please recommend me some MOT algorithms?

**Reply by ZhiyuanChen** [reply]
> Thanks for answering.

No worries

> Actually I'm quite unfamiliar to this kind of network, so if it's ok, would you please recommend me some MOT algorithms?

DeepSort
